Packs/CortexResponseAndRemediation/ReleaseNotes/1_1_77.md

```diff
@@ -0,0 +1,40 @@
+
+#### Playbooks
+
+##### New: Script file added to startup-related Registry keys
+
+This playbook is designed to handle "Script file added to startup-related Registry keys" alerts.
```

Comment:
This playbook handles **Script file added to startup-related Registry keys** alerts.